keras_cv/layers/preprocessing/cut_mix.py

```diff
@@ -33,7 +33,8 @@ class CutMix(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
     ```python
     (images, labels), _ = tf.keras.datasets.cifar10.load_data()
     cutmix = keras_cv.layers.preprocessing.cut_mix.CutMix(10)
-    output = cutmix({'images': images, 'labels': labels})
+    images, labels =  images[:32], labels[:32]
```

Comment:
can you one hot in an extra line and add the same code to the mixup demo/docstring?

So how about:
```
(images, labels), _ = tf.keras.datasets.cifar10.load_data()
labels = tf.one_hot(labels.squeeze(), 10)
cutmix = keras_cv.layers.preprocessing.cut_mix.CutMix(10)
cutmix({"images": images[:32], "labels": labels[:32]})
```

Reply:
> how about:
> 
> ```
> (images, labels), _ = tf.keras.datasets.cifar10.load_data()
> labels = tf.one_hot(labels.squeeze(), 10)
> 
> cutmix = keras_cv.layers.preprocessing.cut_mix.CutMix(10)
> cutmix({"images": images[:32], "labels": labels[:32]})
> ```
> 
> (Please include the newline too!)

I think that works perfectly. Let me make that change.